components/calculation/MeasureHighlightingPanel.tsx

```diff
@@ -68,8 +71,21 @@ export default function MeasureHighlightingPanel({ patientId }: MeasureHighlight
         limit={100}
         label="Search CQL Expression Definition"
         onItemSubmit={item => {
-          document.getElementById(defIds[item.value])?.scrollIntoView({ behavior: 'smooth' });
+          document
+            .querySelector(
+              `pre[data-statement-name="${item.value.split('"')[1]}"[data-library-name="${item.value.split('"')[0]}"]`
```

Comment:
This still needs the `]` for the first part of the selector on the statement name. Currently this makes chrome throw an error. You can see in this case right after `"Denominator"` it needs the `]`

```
Uncaught DOMException: Failed to execute 'querySelector' on 'Document': 'pre[data-statement-name="Denominator"[data-library-name="HIVScreeningFHIR."]' is not a valid selector.
```

Reply:
Also just noticed that you will not want the `.` in the library name.